package abd.abd;

import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.PreparedStatement;
import java.sql.SQLException;
import java.util.Properties;
 
public class App {
    public static void main(String[] args) {
 
        // creates three different Connection objects
        Connection conn = null;
 
        try {
            // connect way #1
            String url = "jdbc:mysql://localhost/banco";
            String user = "root";
            String password = "jig30592";
 
            
            
            conn = DriverManager.getConnection(url, user, password);
            if (conn != null) {
                System.out.println("Connected to the database banco");
            }
            
            
            String query = " insert into cuenta (nCuenta, nombre, apellidos, saldo)"
                    + " values (?, ?, ?, ?, ?)";

          // create the mysql insert preparedstatement
            PreparedStatement preparedStmt = conn.prepareStatement(query);
            preparedStmt.setString (1, "12345678901234567890");
            preparedStmt.setString (2, "Jesus");
          	preparedStmt.setString (3, "Iniguez");
          	preparedStmt.setDouble    (4, 0);
          	
          	preparedStmt.execute();
          	
          	conn.close();
            /*// connect way #2
            String url2 = "jdbc:mysql://localhost:3306/test2?user=root&password=secret";
            conn2 = DriverManager.getConnection(url2);
            if (conn2 != null) {
                System.out.println("Connected to the database test2");
            }
 
            // connect way #3
            String url3 = "jdbc:mysql://localhost:3306/test3";
            Properties info = new Properties();
            info.put("user", "root");
            info.put("password", "secret");
 
            conn3 = DriverManager.getConnection(url3, info);
            if (conn3 != null) {
                System.out.println("Connected to the database test3");
            }
            */
        } catch (SQLException ex) {
            System.out.println("An error occurred. Maybe user/password is invalid");
            ex.printStackTrace();
        }
    }
}
